BSB_lan.ino

```diff
@@ -441,21 +441,41 @@
 #include "BSB_lan_config.h"
 #include "BSB_lan_defs.h"
 
+#if defined(__AVR__)
 #include <avr/pgmspace.h>
-//#include <avr/wdt.h>
+#include <EEPROM.h>
+#include <SPI.h>
+#endif
 #include <Arduino.h>
+
+#if defined(__arm__)
 #include <SPI.h>
- //Mega
-#if defined(__AVR__)
-#include <EEPROM.h>
-#else // __SAM3X8E__
 #include <Wire.h>
 #include "src/I2C_EEPROM/I2C_EEPROM.h"
 template<uint8_t I2CADDRESS=0x50> class UserDefinedEEP : public  eephandler<I2CADDRESS, 4096U,2,32>{};
 // EEPROM 24LC32: Size 4096 Byte, 2-Byte address mode, 32 byte page size
 // EEPROM 24LC16: Size 2048 Byte, 1-Byte address mode, 16 byte page size
 UserDefinedEEP<> EEPROM; // default Adresse 0x50 (80)
 #endif
+
+#if defined(ESP32)
+#include <EEPROM.h>
+#include <ESPmDNS.h>
+EEPROMClass EEPROM("eeprom1", 0x800);
+#define strcpy_PF strcpy
+#define strcat_PF strcat
+#define strchr_P strchr
+#if defined(EEPROM_ERASING_PIN)
+#undef EEPROM_ERASING_PIN
+#define EEPROM_ERASING_PIN 34
+#endif
+#if defined(EEPROM_ERASING_GND_PIN)
+#undef EEPROM_ERASING_GND_PIN
+#endif
```

Comment:
This would need to go together with the EEPROM_ERASING_GND_PIN section mentioned below